aieng-ui/backend/tests/test_acceptance_edit_resolve_compare.py

```
@@ -0,0 +1,274 @@
+"""The one task the project promises, run end to end through the tool surface.
+
+From the 2026-09-06 direction review:
+
+    对一个有明确尺寸、载荷和约束的支架或安装件，完成建模、参数修改、重新分析，
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)